docker/intel-dgpu-driver.Dockerfile

```diff
@@ -37,10 +38,17 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/drm .config \
     && make olddefconfig && make -j $(nproc) && make modules_install
 
+# Create /build/modules directory with all *.ko and *.ko.xz files
+RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -type f | xargs -I {} cp {} /build/modules/ \
+    && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko.xz" -type f | xargs -I {} cp {} /build/modules/ \
```

Comment:
Do we need to package .ko.xz file?
I think we only need to package .ko files OR .ko.xz files.
Since .ko.xz file is compressed ko file to save the space at the expense of loading speed.
For our usage scenario, I prefer to only package the ko file.

Reply:
Good question. We also need the .ko.xz files which are for different kernel modules. Those modules only have the compressed version which is .ko.xz extension. For example, here is the output of `cat modules.dep | grep drm_kms:`

```drm_kms_helper.ko: syscopyarea.ko.xz sysfillrect.ko.xz sysimgblt.ko.xz fb_sys_fops.ko.xz drm.ko dmabuf.ko```

Reply:
After analysis of the dependency tree of i915, it was noted there are several sub dependencies which are in .ko.xz format. As a result, we copy all *.ko.xz files. This will also help ensure in future if a new dependency is added, as we pick up all the files as opposed to a subset. If we were to choose a specific subset of files, then when the dependency tree changes, our build will break and we will need to update our dockerfile.

Reply:
I see. 
for the performance reason, I think in the building stage we should only build ko and not compress it as .ko.xz.
I think It should be an enhancement in the GPU driver project. We can now also add ko.xz files. And at the same time, we can file an issue in the GPU driver project and ask them why some of the files use ko, some others use ko.xz. 
What do you think of it

Reply:
Sounds good